src/include/type/catalog_type.h

```diff
@@ -0,0 +1,77 @@
+//===----------------------------------------------------------------------===//
+//
+//                         Peloton
+//
+// catalog_type.h
+//
+// Identification: src/include/type/catalog_type.h
```

Comment:
@jarulraj We should discuss where to put this file. I think that it should probably be in 'src/include/catalog' not 'src/include/type'. Maybe we should rename it to `catalog_defaults.h` or something else. These are not really "types".